### PR TITLE
Add SimpleComposerHostBot and EchoComposerSkillBot projects

### LINKED ISSUE #179: Create composer version of SimpleBotHost and EchoSkillBot
This issue involves:

- [ ] Implement a SimpleComposerHostBot that does the same thing that SimpleHostBot does.
- [ ] Implement a EchoComposerSkillBot that does the same thing as EchoSkillBot.

Updating pipelines and writing tests for these new bots should be addressed in another issue.

### PR DESCRIPTION
**Pull request by gabog**
Fixes #179 

- Updated SimpleComposerHostBot with ejected runtime code
- Added new EchoComposerSkillBot with ejected runtime
- Added Composer/Directory.Build.props files with excludes for a bunch of violations for the composer ejected runtime (we want to avoid customizing the runtime and these will be addressed in a future version of composer)
- Added temporary ArrangeComposer.ps1 powershell to facilitate rearranging the composer files in the ejected runtime so we can right click publish from the VS (this will need to updated as part of a different issue)

image

As part of this PR we also:

- Updated launchsettings.json on the dotnet bots to use predefined localhost ports to make it easier to run them locally withouth having to reconfigure them.
- Updated default.html pages to show the right names for the bots. 
- Removed botId validation logic from SimpleHostBot and SimpleHostBot-2.1 so we can run the bots locally without needing an AppId and password.

### CODE REVIEW

File: Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/Action/MultiplyDialog.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
+using System;
```
Comment (ceciliaavila) [inline Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/Action/MultiplyDialog.cs:1]: Here the copyright header is missing.

File: Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/CustomActionComponentRegistration.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+using Microsoft.Bot.Builder;
```
Comment (ceciliaavila) [inline Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/CustomActionComponentRegistration.cs:1]: Here the copyright header is missing.

File: Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/tests/ActionsTests.cs [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+using Microsoft.Bot.Schema;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Tests
+{
+    [TestClass]
```
Comment (ceciliaavila) [inline Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/tests/ActionsTests.cs:19]: Should we migrate these tests to Xunit?

File: Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/Action/MultiplyDialog.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
+using System;
```
Comment (ceciliaavila) [inline Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/Action/MultiplyDialog.cs:1]: The copyright header is missing here.

File: Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/CustomActionComponentRegistration.cs [hunk 5]
```diff
@@ -0,0 +1,23 @@
+using Microsoft.Bot.Builder;
```
Comment (ceciliaavila) [inline Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/CustomActionComponentRegistration.cs:1]: The copyright header is missing here.

**Review by ceciliaavila — APPROVED** [review]
Some small comments, but LGTM.

File: Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/tests/ActionsTests.cs (hunk 3 above)
Comment (gabog) [reply Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/tests/ActionsTests.cs:19]: Not for now, this is how composer spits out the runtime, I want to minimize changes to composer to make it easier to maintain.

File: Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/Action/MultiplyDialog.cs (hunk 1 above)
Comment (gabog) [reply Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/Action/MultiplyDialog.cs:1]: Opened an issue with composer reporting this (https://github.com/microsoft/BotFramework-Composer/issues/4923)

File: Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/CustomActionComponentRegistration.cs (hunk 2 above)
Comment (gabog) [reply Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/CustomActionComponentRegistration.cs:1]: Opened an issue with composer reporting this (https://github.com/microsoft/BotFramework-Composer/issues/4923)

File: Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/Action/MultiplyDialog.cs (hunk 4 above)
Comment (gabog) [reply Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/Action/MultiplyDialog.cs:1]: Opened an issue with composer reporting this (https://github.com/microsoft/BotFramework-Composer/issues/4923)

File: Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/CustomActionComponentRegistration.cs (hunk 5 above)
Comment (gabog) [reply Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/CustomActionComponentRegistration.cs:1]: Opened an issue with composer reporting this (https://github.com/microsoft/BotFramework-Composer/issues/4923)

**Review by mrivera-ms — APPROVED** [review]

**Pull request merged**